python/tests/llm/clients/anthropic_bedrock/conftest.py

```diff
@@ -0,0 +1,15 @@
+"""Configuration for AnthropicBedrock tests."""
+
+import pytest
+
+from mirascope.llm.clients.anthropic_bedrock import clients
+
+
+@pytest.fixture(autouse=True)
+def clear_client_cache() -> None:
+    """Clear the LRU cache before and after each test to prevent interference."""
+    clients._anthropic_bedrock_singleton.cache_clear()
```

Comment:
Hmm it's a bit of a smell to me that we need access to the private internals to do this, do you think this is something we'd want public access for?